Assets/Scripts/SS3D/Systems/Entities/Humanoid/HumanoidLivingController.cs

```diff
@@ -33,7 +33,8 @@ protected override void ProcessCharacterMovement()
             ProcessToggleRun();
             ProcessPlayerInput();
 
-            _characterController.Move(Physics.gravity);
+            Physics.SyncTransforms();
```

Comment:
Can you add a line explaining why we need this ? It's really not obvious without a comment. Something similar to the resolution note you linked.